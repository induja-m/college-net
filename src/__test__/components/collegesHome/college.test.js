import College from "../../../components/collegesHome/college"

import renderer from 'react-test-renderer'

const clickFn = jest.fn();

describe('College', () => {

  it('should render with params', ()=>{
    const application = {collegename: "BITS Pilani",course: "ECE",id: 1,status: "Applied",stream: "Engineering",studentcontact: 9876543210,studentgender: "female",studentname: "Emily"};
    let wrapper = renderer.create(<College application={application} key={application.id}/>).toJSON();
    expect(wrapper).toMatchSnapshot();
  })
});