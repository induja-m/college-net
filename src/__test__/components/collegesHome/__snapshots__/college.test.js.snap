// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`College should render with params 1`] = `
<div
  className="row"
  style={
    Object {
      "marginTop": "20px",
    }
  }
>
  <div
    className="col-md-8"
  >
    <div
      className="card"
    >
      <div
        className="card-body"
      >
        <div
          className="card-title h5"
        >
          Applied at 
          BITS Pilani
        </div>
        <p
          className="card-text"
        >
          <i>
            <strong>
              Name: 
            </strong>
            Emily
          </i>
          <p>
             
            <strong>
              Gender: 
            </strong>
            female
          </p>
          <p>
            <strong>
              Course applied: 
            </strong>
            ECE
          </p>
          <p>
            <strong>
              Contact: 
            </strong>
            9876543210
          </p>
        </p>
        <div />
      </div>
    </div>
  </div>
</div>
`;

exports[`College should render without params 1`] = `
<div
  className="row"
  style={
    Object {
      "marginTop": "20px",
    }
  }
>
  <div
    className="col-md-8"
  >
    <div
      className="card"
    >
      <div
        className="card-body"
      >
        <div
          className="card-title h5"
        >
          Applied at 
          BITS Pilani
        </div>
        <p
          className="card-text"
        >
          <i>
            <strong>
              Name: 
            </strong>
            Emily
          </i>
          <p>
             
            <strong>
              Gender: 
            </strong>
            female
          </p>
          <p>
            <strong>
              Course applied: 
            </strong>
            ECE
          </p>
          <p>
            <strong>
              Contact: 
            </strong>
            9876543210
          </p>
        </p>
        <div />
      </div>
    </div>
  </div>
</div>
`;
